.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+## Testing
+
+This app doesn't have CI setup, and it would be misleading to do so:
+the behaviour of the tests changes depending on the environment they
+are run in. **You should manually test your PR before merging.**
+
+- [ ] Tested change in at least one environment
+
+> Example steps to test in Integration:
+>
+> - Click "Configure" for the [Smokey job][]
+> - Change the "Branch specifier" to the name of your branch
+> - Run a new build of the Smokey project and check the results
+
+## Deployment
+
+**You need to manually deploy your PR after merging.**
+
+- [ ] Ran `Smokey deploy` job in all environments
+
+This app is deployed in two ways (see beginning of [the README](README.md)):
+
+- The [Smokey job][] will pick up the changes on `master`.
+- The [continuous Smokey loop](https://github.com/alphagov/govuk-puppet/blob/master/modules/monitoring/templates/smokey-loop.conf) needs manually deploying
```

Comment:
Should we add instructions or a link to how to deploy? Is it the normal process?

Reply:
Or is this covered by the 'deploy smokey' job?

Reply:
I've rewritten this to explain the steps.